.rubocop.yml

```diff
@@ -1,7 +1,20 @@
+require:
+  - rubocop-rspec
+
 inherit_from: .rubocop_todo.yml
 
 AllCops:
   TargetRubyVersion: 3.0
+  SuggestExtensions: false
+  DisplayCopNames: true
+  Include:
+    - './Rakefile' # rake only
+    - '**/*.rb'
+    - '**/*.rake' # rake only
+  Exclude:
+    # autogenerated files (that we don't change) should go here
+    - 'spec/spec_helper.rb'
+    - 'vendor/**/*' # gems only (?)
```

Comment:
Is there a vendor directory?

Reply:
it's a gem and sometimes in deployment there is?    In the distant past, we've had rubocop fail due to stuff in the vendor directory.

Reply:
I don't have a vendor directory, do you? If not, then this is cruft.